client/state/plugins/installed/README.md

```diff
@@ -26,6 +26,8 @@ A module for managing installed plugins on connected sites.
 
 ### `isRequestingForSites( state: Object, sites: Array )`
 
+### `isRequestingForAllSites( state: Object )`
```

Comment:
Let's make sure to document this selector.

Reply:
Thanks! The selector was documented in 839e7834a1f19087233fee5137dc89ec9bb1a5e2